libs/api/domains/driving-license/src/lib/graphql/main.resolver.ts

```diff
@@ -49,4 +58,71 @@ export class MainResolver {
   drivingLicensePenaltyPointStatus(@CurrentUser() user: User) {
     return this.drivingLicenseService.getPenaltyPointStatus(user.nationalId)
   }
+
+  @Query(() => HasTeachingRights)
+  drivingLicenseTeachingRights(@CurrentUser() user: User) {
```

Comment:
TS return types are inconsistently defined in this resolver.
Consider adding return types to all resolve functions as this would ensure this resolver return correct type if changes are made to the underling service

Reply:
yes, this needs a closer look as a followup